src/locales/en.json

```diff
@@ -232,6 +238,12 @@
       "microcovid_budget_one_percent": "Keeps to a budget of 200 microCOVID per week (\"Standard Caution Budget\")",
       "microcovid_budget_deci_percent": "Keeps to a budget of 20 microCOVID per week (\"High Caution Budget\")",
       "hasCovid": "Has COVID"
+    },
+    "vaccine": {
+      "pfizer": "Pfizer BioNTech",
+      "moderna": "Moderna",
+      "astra_zeneca": "AstraZeneca",
+      "unknown": "Unknown or other"
```

Comment:
I'm not sure how well this will be received given the experience with "Other" and masks, but I also don't have a confident suggestion. @octern ?

Reply:
With masks, I know what mask I have and it's not on the list. With vaccines, it's much more likely that a person just doesn't know which one they got.